### DISCUSSION TITLE
Welcome to 4D-Macros Discussions!

### DISCUSSION THREAD

**Discussion by vdelachaux** [4d/4D-Macros#1] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by kirkroyster** [comment]
I tested the macro that converts C_ declarations to var statements.  I noticed that the macro (which runs as soon as opening a method/etc) misses C_ declarations that are on all lines that immediately follow a comment line // ...

**Comment by vdelachaux** [comment]
Hi,
Do you mean that:
```4d
C_LONGINT($l)
// comment
C_TEXT($t)
```
Ignore C_TEXT line?

**Comment by kirkroyster** [comment]
Yes that is what occurred for me.

**Comment by vdelachaux** [comment]
Thanks, I'll look into it as soon as possible.

**Comment by kirkroyster** [comment]
Super thank you.

**Comment by kirkroyster** [comment]
One thing I should clarify, I am using R7 when experiencing this issue.

**Comment by vdelachaux** [comment]
I don't have the issue with the [last release](https://github.com/4d/4D-Macros/releases/tag/1.03), did you try it ?

**Comment by kirkroyster** [comment]
I verified that 1.0.3 updated automatically, and I confirmed that it is converting the lines immediately following the comment lines.  Thank you for reaching out!

**Comment by kirkroyster** [comment]
Also note that I did the above test on R8.
